src/Scriban.Tests/TestParser.cs

```diff
@@ -607,6 +608,17 @@ public static void Doc_timespan(string inputName, string input, string output)
             AssertTemplate(output, input);
         }
 
+
+        [Test]
+        public void EnsureThatItemWithIndexePropertyDoesNotThrow()
+        {
+            var obj = JObject.Parse("{\"name\":\"steve\"}");
```

Comment:
ha, I'm in there for ever as an example of how to break it now :)

Reply:
Oops - sorry! At least you didn't include your full contact details ;-)